website/docs/docs/build/build-semantic-layer-intro.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Build your Semantic Layer"
+id: build-semantic-layer-intro
+description: "Learn about MetricFlow and build your Semantic Layer with Semantic models and Metrics"
+sidebar_label: Build your Semantic Layer
+tags: [Metrics, Semantic Layer]
+hide_table_of_contents: true
+---
+
+Seamlessly build your semantic layer using [semantic models](/docs/build/semantic-models) and [metrics](/docs/build/metrics-overview), avoid repetition, and  optimize your workflow with concepts you're already familiar with using dbt metrics, powered by **MetricFlow**. 
```

Comment:
```suggestion
Seamlessly build your semantic layer using [semantic models](/docs/build/semantic-models) and [metrics](/docs/build/metrics-overview), avoid repetition, and  optimize your workflow with concepts you're already familiar with using dbt metrics, powered by **MetricFlow**. 
```
```suggestion
Build your semantic layer seamlessly using dbt Metrics, powered by **MetricFlow**. Use [semantic models](/docs/build/semantic-models and [metrics](/docs/build/metrics-overview) to avoid repetition and optimize your workflow with concepts you're already familiar with. 
```